docs/_bin/deploy-docs.sh

```diff
@@ -88,6 +88,9 @@ remote=$(git -C "$druid" config --local --get "remote.$origin.url")
 git clone -q --depth 1 --branch $branch $remote "$src"
 
 if [ -n "$opt_docs" ] ; then
+  # Check for broken links
+  "$src/docs/_bin/broken-link-check.py" "$src/docs/content" "$src/docs/_redirects.json"
```

Comment:
As far as I could tell, this script doesn't modify anything and just prints out the broken links/redirects, what is the purpose of calling it here? Should we be catching the exit code and have it cause a failure of some sort if it has non-zero exit code?

Reply:
The bash script is running with mode `-e` so if any subcommand exits nonzero, it will automatically stop.

Reply:
Oops! I see that now, my bad :+1: